erts/emulator/beam/bif.c

```diff
@@ -5275,3 +5275,48 @@ BIF_RETTYPE dt_restore_tag_1(BIF_ALIST_1)
 #endif
     BIF_RET(am_true);
 }
+
+BIF_RETTYPE map_get_2(BIF_ALIST_2) {
```

Comment:
Why duplicate so much code?

I suggest that you make a copy of `maps_get_2()` in `erl_map.c` and rename it to `map_get_2()`.